app/hooks/handle_send_message.ts

```diff
@@ -119,12 +121,22 @@ export const useHandleSendMessage = ({
             response = await createPost(serverUrl, post, postFiles);
         }
 
+        if (response.error) {
```

Comment:
do we expect to show this snackbar even for posts that are not scheduled?

Reply:
Nice catch. This should only check for the error if sending from the draft view. Will update the code.